.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
As asked in the requirement, please add in the .gitignore file the way to ignore test.md

<img width="643" alt="image" src="https://user-images.githubusercontent.com/7243132/177001250-e6392148-677b-476a-8c5a-bd49606ff6a4.png">

Reply:
- [ ] Just as suggested by the previous reviewer, you should have a `test.md` file with the content `This file should be ignored by git` and add the file to `.gitignore` just as you did for the `node_modules`

Example:
```
node_modules/
test.md
```